rules/opentitan.bzl

```diff
@@ -351,30 +356,88 @@ bin_to_spiflash_frames = rule(
     },
 )
 
+def _gen_sim_dv_logs_db_impl(ctx):
+    outputs = []
+    for src in ctx.files.srcs:
+        if src.extension != "elf":
+            fail("can only generate DV logs database files from ELF files.")
+        logs_db = ctx.actions.declare_file("{}.logs.txt".format(
```

Comment:
The extension of this file seem like something we'd use for logging output more than enumeration of logging points in memory and their formatting. I think it's baked in enough for now that it's fine not to chase it all the way down. In most of the places you handle this you're making it clear your generating something like a database of logging structures.

Reply:
Yeah this is buried all the way down to the testbench.